src/main/java/formflow/library/email/MailgunEmailClient.java

```diff
@@ -19,9 +19,21 @@
 @Slf4j
 public class MailgunEmailClient implements EmailClient<MessageResponse> {
 
+  /**
+   * String required for sending email
+   */
   private String senderEmail;
+  /**
+   * String storing the Mailgun domain
+   */
   private final String mailgunDomain;
+  /**
+   * Mailgun messages api object
+   */
   private MailgunMessagesApi mailgunMessagesApi;
+  /**
+   * Boolean variable sets requireTLS to true.
+   */
```

Comment:
Do we need to document private variables? I doubt these show up in the javadocs...?

Reply:
Not that it hurts to document the, but I don't think we _have_ to.

Reply:
Do you have any thoughts on this?  I'm fine keeping it in, just wondering if you knew of a best practice around this?

Reply:
Documenting private variables is not included in the javadocs warnings for missing comments